packages/kr.rb

```diff
@@ -0,0 +1,54 @@
+require 'package'
+
+class Kr < Package
+  description 'Krypton command-line client, daemon, and SSH integration'
+  homepage 'https://krypt.co'
+  version '2.4.4'
+  source_url 'https://raw.githubusercontent.com/kryptco/kr/master/README.md'
+  source_sha256 'a0452783d22218a6fab914abbf5263459aaa1c8745512d7c4ebcf51fe08aaf7a'
+
+  depends_on "rust" => :build
+  depends_on "go" => :build
+  depends_on "libsodium" => :build
+  depends_on "rsync" => :build
+  depends_on "psmisc" => :build
+
+  def self.install
+    cargo_web_sha256 = "c8a92ed0185206ad7f2da8e1692d168b20044de7066b2eb086e4bbb0d06ff787"
+    ENV["GOPATH"] = "#{Dir.pwd}/go"
+    ENV["RUSTUP_HOME"] = "#{Dir.pwd}/.rustup"
+    ENV["CARGO_HOME"] = "#{Dir.pwd}/.cargo"
+    system "mkdir -p #{ENV["RUSTUP_HOME"]}"
+    system "mkdir -p #{ENV["CARGO_HOME"]}"
+
+    ENV["PATH"] = "#{ENV["CARGO_HOME"]}/bin:/usr/local/bin:/usr/bin:/bin:/opt/bin"
+
+    system "rustup", "install", "stable"
+    system "rustup", "default", "stable"
+    system "rustup", "target", "add", "wasm32-unknown-emscripten"
+
+    system "cargo web --version || ( curl -Ls https://github.com/koute/cargo-web/archive/0.6.10.zip -o cargo-web.zip; if [ $(sha256sum cargo-web.zip | cut -d ' ' -f1) != '#{cargo_web_sha256}' ]; then echo 'cargo-web sha did not match #{cargo_web_sha256}, exiting.'; exit 1; fi; unzip cargo-web.zip; cd cargo-web-0.6.10; cargo build --release && cargo install )"
```

Comment:
Can this long line be split into separate lines for readability?

Reply:
Yes, I can look at switching it to heredoc. I had it this way so that it could be skipped in the case that you had it already but that is unlikely now with it sandboxed so likely this can just be all separate system calls now.

Reply:
Done